specification/schemas/services/Service.json

```diff
@@ -146,6 +146,12 @@
             },
             "service_options": {
               "$ref": "#/components/schemas/ServiceOptionsRelationship"
+            },
+            "zone_from": {
+              "$ref": "#/components/schemas/ZoneRelationship"
+            },
+            "zone_to": {
+              "$ref": "#/components/schemas/ZoneRelationship"
             }
```

Comment:
Since we expose these relationships, we should also unhide the `GET /zones` and `GET /zones/{zone_id}` routes (remove a line from the javascript file).

Reply:
They aren't hidden though, just at the bottom lol